service-workers/service-worker/fetch-event.https.html

```diff
@@ -851,5 +851,27 @@
     assert_equals(frame.contentDocument.body.textContent,
                   'method = POST, isHistoryNavigation = true');
   }, 'FetchEvent#request.isHistoryNavigation is true (POST + history.go(-1))');
+
+promise_test(async t => {
+    // Set page_url to "?ignore" so the service worker falls back to network
+    // for the main resource request, and add a suffix to avoid colliding
+    // with other tests.
+    const page_url = 'resources/?ignore-for-request-body-fallback-string';
+
+    const frame = await with_iframe(page_url);
+    t.add_cleanup(() => { frame.remove(); });
+    // Add "?clone-and-ignore" so the service worker falls back to
+    // echo-content.py.
+    const echo_url = '/fetch/api/resources/echo-content.py?status=421';
+    const response = await frame.contentWindow.fetch(echo_url, {
+        method: 'POST',
+        body: 'text body'
+    });
+    const text = await response.text();
+    assert_equals(
```

Comment:
Let's also assert the status on the response.

Reply:
Done.